server/modules/storage/s3/common.js

```diff
@@ -0,0 +1,64 @@
+const S3 = require('aws-sdk/clients/s3')
+
+/* global WIKI */
+
+/**
+ * Deduce the file path given the `page` object and the object's key to the page's path.
+ */
+const getFilePath = (page, pathKey) => {
+  const fileName = `${page[pathKey]}.${page.getFileExtension(page.contentType)}`
```

Comment:
`getFileExtension()` shouldn't have any argument.

Reply:
goot spot! addressed in 8861944727cf15d5022f10aa450f0f6385afcd5d